index.js

```diff
@@ -51,10 +51,13 @@ module.exports = function(content) {
 	links.reverse();
 	var data = {};
 	content = [content];
+	function processOther(link) {
```

Comment:
I think the function and options name should be something line `processURI` . right !!